pkg/component/observability/monitoring/prometheus/cortex.go

```diff
@@ -77,7 +77,7 @@ func (p *prometheus) cortexVolume(configMapName string) corev1.Volume {
 func (p *prometheus) cortexConfigMap() *corev1.ConfigMap {
 	obj := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      p.name() + "-" + configMapNamePrefix,
+			Name:      p.name() + "-" + cortexConfigNameInfix,
```

Comment:
Nit: Is it really an infix? At least here it still looks like a suffix, but the name may be used in other ways.

Otherwise, looks good.

/lgtm

Reply:
Ref https://github.com/gardener/gardener/pull/9606#discussion_r1572233622 :)

Reply:
Ok, works for me.